safe/definitions/osm_downloader.py

```diff
@@ -0,0 +1,7 @@
+# coding=utf-8
+"""Definitions relating to the OSM Downloader."""
+
+PRODUCTION_SERVER = 'http://osm.inasafe.org/'
+STAGING_SERVER = 'http://78.46.206.68:8080/'
```

Comment:
Is it ok to have this?
Perhaps put it in the options dialog to set prod and staging server

Reply:
It's only visible if you have dev mode. It's only a placeholder